omlmd/helpers.py

```diff
@@ -27,20 +26,18 @@ def download_file(uri: str):
     return file_name
 
 
+@dataclass
 class Helper:
-    _listeners: list[Listener] = []
-
-    def __init__(self, registry: OMLMDRegistry | None = None):
-        if registry is None:
-            self._registry = OMLMDRegistry(
-                insecure=True
-            )  # TODO: this is a bit limiting when used from CLI, to be refactored
-        else:
-            self._registry = registry
+    registry: OMLMDRegistry = (
+        field(  # TODO: this is a bit limiting when used from CLI, to be refactored
+            default_factory=lambda: OMLMDRegistry(insecure=True)
+        )
+    )
+    _listeners: list[Listener] = field(default_factory=list)
 
-    @property
-    def registry(self):
-        return self._registry
+    @classmethod
+    def from_plain(cls, insecure: bool):
```

Comment:
any suggestions for what this constructor might be named?

Reply:
in case, what about `from_default_registry` ?